resources/dicts/patrols/general/border.json

```diff
@@ -182,6 +182,29 @@
     "history_text": [
         "This cat got a scar from a fight with a small dog, but they always say it was a huge dog."
     ]
+},
+{
+    "patrol_id": "gen_bord_abandonedkittypet1",
+    "biome": "Any",
+    "season": "Any",
+    "tags": ["border", "new_cat_injury_kittypet", "nc_sickness",  "respect", "warrior"],
+    "intro_text": "The patrol hears the wailing of a cat and the sound of a monster speeding away.",
+    "decline_text": "It's none of their business really and they have to finish the border patrol.",
```

Comment:
Feels a little awkward, could reword to:
"They decide it's none of their business and turn to continue the border patrol."